src/react-native-animated-charts/src/charts/linear/ChartLabels.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { TextInput, TextInputProps } from 'react-native';
+import Animated, { useAnimatedProps } from 'react-native-reanimated';
+import { useChartData } from '../../helpers/useChartData';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+interface ChartLabelProps extends TextInputProps {
+  format: (value: string) => string;
+}
+
+const ChartLabelFactory = (fieldName: 'originalX' | 'originalY') => {
+  const ChartLabel: React.FC<ChartLabelProps> = ({ format, ...props }) => {
+    const chartData = useChartData();
+    const val = chartData[fieldName];
+
+    const textProps = useAnimatedProps(
+      () => ({
+        text: format ? format(val.value) : val.value,
```

Comment:
```suggestion
        text: format?.(val.value) ?? val.value,
```
similarly, line below

Reply:
Hermes didn't work properly with these things. It's good it does. But tbh I don't like the cryptic look of `format?.()`